index.html

```diff
@@ -127,12 +127,12 @@ <h4 class="c1dp deskim">Uber Navigation</h4>
             <li class="special">JavaScript</li>
           </ul>
         </div>
-        <button class="efecbot">See project</button>
+        <button class="efecbot showpop">See project</button>
       </div>
     </div>
     <div class="card3 cg">
-      <img src="Img/Snapshoot Portfolio 3.png" alt="card 3" class="mobim">
-      <img src="Img/Snapshoot Portfolio desk.png" alt="card desk" class="deskim">
+      <img src="Img/SnapshootPortfolio3.png" alt="card 3" class="mobim">
+      <img src="Img/SnapshootPortfoliodesk.png" alt="card desk" class="deskim">
```

Comment:
Nice work on implementing the functionalities of the popup window and creating the array of object containing project information as required👍. but don't forget that it is a requirement in this milestone that the popup and project section be generated dynamically making use of the data from the created array👍. Kindly ensure these sections are not hardcoded by taking them away from your `index.html` file👍.

<img width="817" alt="popup instruction" src="https://github.com/AndreaM2429/Portfolio_setup_and_mobile_first/assets/64643995/5b28bb71-d70d-484d-a366-52cdd8682879">